src/status_im2/contexts/wallet/receive/style.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im2.contexts.wallet.receive.style
+  (:require [quo.foundations.colors :as colors]))
+
+(def blur
+  {:position      :absolute
```

Comment:
same here, afaik this is the overlay of :type :shell being used here -> 

<img width="319" alt="Screenshot 2023-12-18 at 13 29 47" src="https://github.com/status-im/status-mobile/assets/22799766/f29f118e-bf06-452a-9730-2836f68d1e13">